HederaApi.cmake

```diff
@@ -1,5 +1,5 @@
-set(HAPI_LIBRARY_HASH "988a6c8040d9ec21ace8f297929a76bc25710462de836e7fc3218a5d01ab7a1b" CACHE STRING "Use the configured hash to verify the Hedera API protobuf library release")
-set(HAPI_LIBRARY_URL "https://github.com/hashgraph/hedera-protobufs-cpp/releases/download/v0.47.0/hapi-library-5dabd1b5.tar.gz" CACHE STRING "Use the configured URL to download the Hedera API protobuf library package")
+set(HAPI_LIBRARY_HASH "e0fb2bd142f315c3062ced9850823a452dd9d0c80cd6a5691c68833a99bf60d2" CACHE STRING "Use the configured hash to verify the Hedera API protobuf library release")
+set(HAPI_LIBRARY_URL "https://github.com/hashgraph/hedera-protobufs-cpp/releases/download/v0.49.0-alpha0/hapi-library-526c8ffd.tar.gz" CACHE STRING "Use the configured URL to download the Hedera API protobuf library package")
```

Comment:
Once I get the Android build figured out for C++ protobufs, I was going to tag `v0.49.0` there. Would we be able to update this after I get that tagged? Or we can do it later, either way

Reply:
I can just wait for you to get the next tag out.

Reply:
https://github.com/hashgraph/hedera-protobufs-cpp/releases/tag/v0.49.0 tagged